code/__DEFINES/mobs.dm

```diff
@@ -128,6 +128,7 @@
 #define TK_USER			(1<<8)
 #define CANUNCONSCIOUS	(1<<9)
 #define CANCONFUSE		(1<<10)
+#define UNDETECTABLE	(1<<10) // Whether not this unit should be detectable by automated means (like turrets). Used by hivemind
```

Comment:
```suggestion
#define UNDETECTABLE	(1<<11) // Whether not this unit should be detectable by automated means (like turrets). Used by hivemind
```
Maybe something like `INVISIBLE` ?
Undetectable is a bit broader.

Reply:
I had thought about that, but undetectable is what i wanted to go with, for example maybe having a xeno that is completely visible but can't be detected on via computers.

Reply:
Then `UNDETECTABLE_BY_MACHINE_`, then?
The flag value needs to change, to be different than `CANCONFUSE`